ofmt/bin/ofmt.js

```diff
@@ -11,19 +11,20 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url))
 const packageRoot = path.resolve(__dirname, '..')
 
 const prettierConfigPath = [
-  'node_modules/@ottofeller/prettier-config-ofmt/index.json', // Path within this package.
   '../prettier-config-ofmt/index.json', // Path within another package, where 'ofmt' is installed as a dependency.
+  'node_modules/@ottofeller/prettier-config-ofmt/index.json', // Path within this package.
```

Comment:
@quesabe why did you have to change the order in which config files are visited?